easybuild/easyconfigs/b/berkeley_upc/berkeley_upc_translator-2.20.2-GCC-4.9.2.eb

```diff
@@ -0,0 +1,33 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+easyblock = 'ConfigureMake'
+
+name = 'berkeley_upc_translator'
+version = '2.20.2'
+
+homepage = 'http://upc.lbl.gov/'
+description = """Unified Parallel C (UPC) is an extension of the C programming
+language designed for high performance computing on large-scale parallel
+machines.The language provides a uniform programming model for both shared and
+distributed memory hardware. The programmer is presented with a single shared,
+partitioned address space, where variables may be directly read and written by
+any processor, but each variable is physically associated with a single
+processor. UPC uses a Single Program Multiple Data (SPMD) model of computation
+in which the amount of parallelism is fixed at program startup time, typically
+with a single thread of execution per processor."""
+
+toolchain = {'name': 'GCC', 'version': '4.9.2'}
+
+block = 'ConfigureMake';
+parallel = 1
+
+source_urls = ['http://upc.lbl.gov/download/release/']
+sources = [SOURCELOWER_TAR_GZ]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': [ 'targ' ]
```

Comment:
no need for the extra spaces before/after `]`/`[`

also, maybe make this a bit more specific? what's in `targ`?